src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -530,32 +531,16 @@ def test_WINNF_FT_S_PCR_2(self, config_filename):
   def generate_PCR_3_default_config(self, filename):
     """Generate the WinnForum configuration for PCR 3."""
     # File path where SAS UUT claimed ppa boundary generated in PCR.1 test
+    pcr_1_test_config_file_name = 'default.config'
     pcr_1_test_config_file_path = os.path.join('testcases', 'configs',
                                                'test_WINNF_FT_S_PCR_1',
-                                               'default.config')
-    pcr_1_test_config_file_name = pcr_1_test_config_file_path.split('/')[-1]
+                                               pcr_1_test_config_file_name)
     sas_uut_claimed_ppa_boundary_file_path = getSasUutClaimedPpaBoundaryFilePath(
         pcr_1_test_config_file_name)
 
-    # Load SAS UUT claimed ppa boundary and check if any error while retrieving
-    # SAS UUT claimed ppa boundary generated in PCR.1 test.
-    try:
-      with open(sas_uut_claimed_ppa_boundary_file_path, 'r') as claimed_ppa_file:
-        user_claimed_ppa_contour = json.load(claimed_ppa_file)
-    except IOError:
-      raise RuntimeError('ConfigError:There is an error in reading path:%s \n'
-                         'Please ensure to execute PCR.1 test first if not already.\n'
-                         % sas_uut_claimed_ppa_boundary_file_path)
-
-    # Shrink the user claimed ppa boundary by 1 kilometer.
-    user_claimed_ppa_contour_shapely = utils.ToShapely(
-        user_claimed_ppa_contour['features'][0]['geometry']).buffer(-1e-2)
-    user_claimed_ppa_contour_geometry = json.loads(utils.ToGeoJson(
-        user_claimed_ppa_contour_shapely))
-
     config = {
         'configPCR_1': pcr_1_test_config_file_path,
-        'userClaimedPpaContour': user_claimed_ppa_contour_geometry
+        'userClaimedPpaContourFilePath': sas_uut_claimed_ppa_boundary_file_path
```

Comment:
This should take the provided contour itself, *not* the contour the came from PCR.1. ITS needs to be able to test with arbitrary contours, not just contours which are slightly smaller than the one created by the SAS UUT. Please undo this most recent change: your previous code was correct to use `'userClaimedPpaContour': user_claimed_ppa_contour_geometry`.

Reply:
We have   moved to assert the contour file is not found raise an exception for PCR 3   test. So repharse the error message and revert it back previous state